pages/deployment/multicontainer.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Multicontainer Guide
+excerpt: A guide to building multicontainer applications with resin.io
+---
+
+# Multicontainer Guide
+
+As your applications grow more complex, you may find significant benefit in running some services in separate containers. Multicontainer applications allow you to better isolate, duplicate, and manage key services, making your applications more modular and secure. Starting with resinOS vx.x.x, resin.io provides multicontainer application support, built around the [Docker Compose][docker-compose] tool. This guide will cover the considerations you need to take into account when using a multicontainer instance, including Docker Compose configuration, important resin.io specific settings, and single container applications.
+
+## Docker Compose
+
+At the root of your multicontainer application, you'll use a `docker-compose.yml` file to specify the configuration of your containers. The `docker-compose.yml` defines the services you'll be building, as well as how the services interact with each other and the host OS.
```

Comment:
Can we briefly explain what docker compose is, what version(s) we support and link to the main docker compose documentation?